### PR TITLE
Update Add-TeamViewerManager.ps1

### PR DESCRIPTION
**Pull request by OtterKring**
included adding UserGroups as manager.

### CODE REVIEW

**Comment by danieljoos** [comment]
💙

Good to see that the Powershell module gets someone to work on 😄.

Would be great if you could also update the docs:

```
Import-Module .\TeamViewerPS
Update-MarkdownHelp .\docs\commands
```

Revert all changes except those to `Add-TeamViewerManager` docs. Then document the new CLI attributes.

Greetings to the team ;-)

**Comment by OtterKring** [comment]
In der Not frißt der Teufel Fliegen! :-P

Where does the Update-MarkdownHelp cmdlet come from?
Is it updating the docs automatically? I just added one parameter. Shouldn't I just add it manually?

**Comment by OtterKring** [comment]
In der Not frißt der Teufel Fliegen. ;-P

Ok, updated.

**Comment by danieljoos** [comment]
lgtm (but I can't merge anylonger). @koutsop

**Review by koutsop — APPROVED** [review]

**Comment by koutsop** [comment]
Hi @danieljoos,

Unfortunately, I don't have write access, thus I cannot approve and run the workflow.  @ChristianJ-TV do you may have write access?

**Comment by MarcelRaad** [comment]
@koutsop You and @ChristianJ-TV have write access now :-)

File: TeamViewerPS/Public/Add-TeamViewerManager.ps1 [hunk 1]
```diff
@@ -77,7 +77,7 @@ function Add-TeamViewerManager {
             $body["id"] = $Manager | Resolve-TeamViewerManagerId
         }
         '*ByUserObject' {
-            $body["accountId"] = $User.Id.TrimStart('u')
+            $body["accountId"] = $User.TrimStart('u')
```
Comment (danieljoos) [inline TeamViewerPS/Public/Add-TeamViewerManager.ps1:80]: I think this wasn't correct in the first place - and also isn't quite correct now.
Could be replaced by something like this:

```powershell
$body["accountId"] = ($User | Resolve-TeamViewerUserId).TrimStart('u')
```

**Comment by OtterKring** [comment]
I added the Remove-TeamViewerUser function. Function, Test and Help .md are based on Remove-TeamViewerGroup.

**Comment by danieljoos** [comment]
To me the `Remove-TeamViewerUser` should go into its own PR.

**Pull request merged**